src/prpy/controllers/ForceTorqueSensorController.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import rospy
+from geometry_msgs.msg import WrenchStamped
+from . import OrController
+
+
+class ForceTorqueSensorController(OrController):
+    def __init__(self, namespace, controller_name, simulated=False):
+        self.logger = logging.getLogger(__name__)
+        self.namespace = namespace
+        self.controller_name = controller_name
+        self.simulated = simulated
+
+        self.logger.info("Force/Torque Sensor Controller initialized")
```

Comment:
Add `namespace` or `controller_name` to this message to differentiate between the left and right arms.